tests/unit/utils/test_utils_tasks.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Zenodo.
+# Copyright (C) 2018 CERN.
+#
+# Zenodo is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Zenodo is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Zenodo; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Test utility tasks."""
+
+from __future__ import absolute_import, print_function
+from zenodo.modules.utils.tasks import file_integrity_report
+from mock import mock
+
+
+def test_file_integrity_report(app, db, test_object):
+    # test without incdent
+    with mock.patch('zenodo.modules.snow.snow_api.SNOW.create_incident') \
+            as mocked:
+        file_integrity_report.apply()
+        mocked.assert_not_called()
+
+    # change the state
+    test_object.file.last_check = False
+    db.session.commit()
+
+    # test with incident
+    with mock.patch('zenodo.modules.snow.snow_api.SNOW.create_incident') \
+            as mocked:
+        file_integrity_report.apply()
+        mocked.assert_called()
+        #  print mock.call_args[1]
```

Comment:
I guess this can be removed or the test can be extended to assert for the passed arguments as well